Dockerfile

```diff
@@ -39,5 +39,6 @@ RUN sed -i.orig -e '/user[[:space:]]\+nginx/d' -e 's@pid[[:space:]]\+.*@pid /tmp
     diff -u /etc/nginx/nginx.conf.orig /etc/nginx/nginx.conf ||: && \
     chown nginx /usr/share/nginx/html/index.html && \
     chown -Rc nginx /var/cache/nginx
-USER nginx
+# USER nginx

```

Comment:
```diff
-# USER nginx
+# Equivalent to 'USER nginx', see: https://github.com/InseeFrLab/onyxia-web/pull/279
```

Reply:
Hi @sathieu,  
Thanks for the PR.  
Please accept my suggestion and I will merge.